quattor/types/hardware.pan

```diff
@@ -243,6 +243,13 @@ type structure_bios = {
     "releasedate" : string
 };
 
+@documentation{
+    Configuration settings
+}
+type structure_hw_options = {
+    "encrypt_disks" ? boolean
```

Comment:
what does this mean? encrypt some/all disk?

Reply:
Yes, encrypt all disks. Would `encrypt_all_disks` be a better name?

Reply:
a docstring is enough for me

Reply:
Done